File: 43- হিসাবকিতাব.c
#include<stdio.h>
int main()
{

    int t;
    scanf("%d",&t);
    while(t--)
    {
        int p,q,c,t=1;
        scanf("%d%d%d",&p,&q,&c);
        for(int i=1;i<=q;i++)
        {
            t=(t*p)%c;
        }
        printf("Result = %d\n",t);

    }
    return 0;


}

File: 43- হিসাবকিতাব.py
t = int(input())

import math

while t!=0:
    p,q,c = input().split()
    p = int(p)
    q = int(q)
    c = int(c)

    s = pow(p, q)
    s = s%c

    print("Result =",s)

    t-=1
